storage/blob_test.go

```diff
@@ -37,10 +37,10 @@ func (s *StorageBlobSuite) Test_pathForBlob(c *chk.C) {
 }
 
 func (s *StorageBlobSuite) Test_blobSASStringToSign(c *chk.C) {
```

Comment:
Already existed in the TestGetBlobSASURIWithSignedIPAndProtocolValidAPIVersionPassed test, or did you want something else?